recipes-bsp/optee-os/optee-os-iot2050_3.21.0.bb

```diff
@@ -28,7 +28,11 @@ OPTEE_EXTRA_BUILDARGS:append:secureboot = " \
     "
 
 # RPMB key pairing
-OPTEE_EXTRA_BUILDARGS:append:rpmb-setup = " CFG_RPMB_WRITE_KEY=y"
+DEPENDS:append:rpmb-setup = " edk2-standalonemm-rpmb"
+DEBIAN_BUILD_DEPENDS:append:rpmb-setup = ", edk2-standalonemm-rpmb"
+OPTEE_EXTRA_BUILDARGS:append:rpmb-setup = " CFG_RPMB_WRITE_KEY=y \
+    CFG_STMM_PATH=/usr/lib/edk2/BL32_AP_MM.fd \
```

Comment:
Extract the commong uefi config bits and inject them via a variable?

Reply:
Added a new "uefi" override, and put the StMM into this override. Both rpmb-setup.yml and secureboot.yaml declare this new override.